app/_tutorials/add-rate-limiting-tiers-with-kong-gateway.md

```diff
@@ -102,3 +155,20 @@ Consumers that are not in a consumer group default to the Rate Limiting advanced
    deck gateway sync deck_files
    ```
 
+1. Run the following command to test the rate limiting as the consumer:
+
+  ```sh
+  for _ in {1..6}; do curl -i http://localhost:8000/example_route -H 'apikey:example_key'; echo; done
+  ```
```

Comment:
I'm not sure what this does, is there a way I should modify it so that we can test each tier? And maybe a user who isn't in a tier?

Reply:
this needs to be updated, I think this comes from the RL on a consumer tutorial. The idea there was to test that a specific consumer reached the quota and then received 429 afterwards.

It needs to be adjusted to this particular tutorial, you can try with at least one tier